src/transformers/data/__init__.py

```diff
@@ -16,6 +16,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from .data_collator import (
+    DataCollatorForLanguageModeling,
+    DataCollatorForPermutationLanguageModeling,
+    DataCollatorForSeq2Seq,
+    DataCollatorForSOP,
+    DataCollatorForTokenClassification,
+    DataCollatorForWholeWordMask,
+    DataCollatorWithPadding,
+    default_data_collator,
+)
```

Comment:
This part is not directly relevant to the current documentation but is necessary for "data_collator" to be recognized properly as a submodule of transformers.data
I don't think it's a problem to have it here.